src/distributions/dragonwell/installer.ts

```diff
@@ -0,0 +1,173 @@
+import * as core from '@actions/core';
+import * as tc from '@actions/tool-cache';
+
+import fs from 'fs';
+import path from 'path';
+
+import {JavaBase} from '../base-installer';
+import {extractJdkFile, getDownloadArchiveExtension} from '../../util';
+import {IDragonwellVersions, IDragonwellAllVersions} from './models';
+import {
+  JavaDownloadRelease,
+  JavaInstallerOptions,
+  JavaInstallerResults
+} from '../base-models';
+
+export class DragonwellDistribution extends JavaBase {
+  constructor(installerOptions: JavaInstallerOptions) {
+    super('Dragonwell', installerOptions);
+  }
+
+  protected async findPackageForDownload(
+    version: string
+  ): Promise<JavaDownloadRelease> {
+    if (!this.stable) {
+      throw new Error('Early access versions are not supported');
+    }
+    let majorVersion = version;
+    if (version.includes('.')) {
+      const splits = version.split('.');
+      majorVersion = splits[0];
+      version = splits.length >= 3 ? splits.slice(0, 3).join('.') : version;
+    }
+    const edition = majorVersion == '17' ? 'Standard' : 'Extended';
+    const availableVersions = await this.getAvailableVersions();
+    const matchVersions = availableVersions
+      .filter(item => item.jdk_version == version && item.edition == edition)
+      .map(item => {
+        return {
+          version: item.jdk_version,
+          url: item.download_link
+        } as JavaDownloadRelease;
+      });
+    const resolvedVersion = matchVersions.length > 0 ? matchVersions[0] : null;
+    if (!resolvedVersion) {
+      const versionsMsg = core.isDebug()
+        ? ' Available versions: ${availableVersions}'
```

Comment:
```suggestion
        ? `Available versions: ${availableVersions}`
```
Template literals are missing.